cal/cal/static/js/services.js

```diff
@@ -526,4 +526,58 @@ analyticsApp.service('QueryService', ['$http', '$q', function($http, $q) {
 
     return [slope, intercept, rSquare];
   }
+
+  this.getDataForCalendarIds = function(type, start, end, calendarId, cache, cacheKey) {
```

Comment:
This should be `getDataForCalendarId` (singular) since this applies to only one calendar.